packages/entities/entities-shared/src/components/entity-base-table/EntityBaseTable.vue

```diff
@@ -239,6 +240,22 @@ const emit = defineEmits<{
 
 const { i18n: { t } } = composables.useI18n()
 
+const tableErrorState = computed((): { hasError: boolean, title?: string, message?: string } => {
+  if (typeof props.errorMessage === 'string') {
+    return {
+      hasError: !!props.errorMessage,
+      title: props.errorMessage,
+      message: undefined,
+    }
+  } else {
+    return {
+      hasError: !!props.errorMessage,
+      title: props.errorMessage?.title,
+      message: props.errorMessage?.message,
+    }
+  }
```

Comment:
IMO this interface is confusing; how does each component know whether to utilize a `string` or the `object` interface? Shouldn't we standardize on one or the other?

Reply:
This design is for the backward compatibility, I don't want introduce a breaking change here because I'm not sure if we have some project using `<EntityBaseTable />` directly